File: src/procedures/models/navigation/PositionMarker.java
package procedures.models.navigation;

import led.ColorRGB;

public class PositionMarker {
	public String mTag = "";
	public float mXPosition = 0.0f;
	public int mPriority = 0;
	public Size mMarkerSize = Size.Single;
	public Attenutation mAttenuation = Attenutation.Disabled;
	public ColorRGB mColor = ColorRGB.white;
}

File: src/procedures/models/navigation/PositionMarker.kt
package procedures.models.navigation

import procedures.models.navigation.Attenutation
import led.ColorRGB

class PositionMarker {
    var mTag = ""
    var mXPosition = 0.0f
    var mPriority = 0
    var mMarkerSize = Size.Single
    var mAttenuation = Attenutation.Disabled
    var mColor = ColorRGB.white
}